### PR TITLE
Add Kava Pioneer Link to Desktop & Mobile

### PR DESCRIPTION
**Pull request by adamrturman**
:zap: _Don't forget to link the Shortcut ticket ID in the branch name!_

## What Changes

- Added link to Kava Pioneer program to desktop left nav & bottom of mobile main page

## Nits

On desktop, sometimes this link gets pushed so far down that it isn't visible without scrolling. It should start flush with the bottom and always be visible without scrolling unless the content is expanded.

### CODE REVIEW

File: src/theme/DocSidebar/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+:root {
+  --collapse-button-bg-color-dark: #2e333a;
+}
+
+.sidebarWrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```
Comment (adamrturman) [inline src/theme/DocSidebar/styles.module.css:17]: Only addition in this file.

File: src/theme/DocSidebar/index.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+              )}
+            >
+              <DocSidebarItems items={sidebar} activePath={path} level={1} />
+            </ul>
+          </div>
+          <PioneerLink />
+        </div>
+      </nav>
+      {hideableSidebar && <HideableSidebarButton onClick={onCollapse} />}
+    </div>
```
Comment (adamrturman) [inline src/theme/DocSidebar/index.js:99]: Only change in this file is to wrap the `ul` with a `div`, and then wrap that and the `PioneerLink` in a `div` so we can use flexbox to push it to the bottom of the left nav.

File: src/theme/DocPage/index.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+          <DocPageContent
+            currentDocRoute={currentDocRoute}
+            versionMetadata={versionMetadata}
+            sidebarName={sidebarName}
+          >
+            {renderRoutes(docRoutes, {
+              versionMetadata,
+            })}
+          </DocPageContent>
+          {windowSize === 'mobile' && <PioneerLink />}
```
Comment (adamrturman) [inline src/theme/DocPage/index.js:178]: Only change in this file is to add the `PioneerLink` to the mobile view of this component.

File: src/theme/DocPage/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+:root {
```
Comment (adamrturman) [inline src/theme/DocPage/styles.module.css:8]: No edits made to this file - the stylesheets get ejected along with the corresponding component.

**Review by downing034 — APPROVED** [review]
Lgtm. Tested on mobile and pioneer link works as expected

**Pull request merged**